code/__DEFINES/shuttles.dm

```diff
@@ -55,6 +55,12 @@
 
 //Docking error flags
 #define DOCKING_SUCCESS 0
-#define DOCKING_BLOCKED 1
-#define DOCKING_IMMOBILIZED 2
-#define DOCKING_AREA_EMPTY 4
\ No newline at end of file
+#define DOCKING_COMPLETE 1
```

Comment:
No such thing as `DOCKING_COMPLETE` in Oracle code. I don't know what you fucked up, but you fucked something up.

❤️

Reply:
https://github.com/tgstation/tgstation/pull/30261/files

It's in /tg/code two days after we forked from them.